api/database/migrations/2022_03_02_214626_create_experience_skill_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateExperienceSkillTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('experience_skills', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->text('details')->nullable();
+            $table->uuid('skill_id')->nullable(false);
+            $table->foreign('skill_id')->references('id')->on('skills');
+            $table->uuid('experience_id')->nullable(false);
+            $table->string('experience_type')->nullable(false);
+            $table->unique(['skill_id', 'experience_id']);
```

Comment:
This should probably include experience_type as well, since id+type identifies an entity in a polymorphic table
```suggestion
            $table->unique(['skill_id', 'experience_id', 'experience_type']);
```